src/sentry/uptime/subscriptions/subscriptions.py

```diff
@@ -299,14 +336,18 @@ def disable_project_uptime_subscription(uptime_monitor: ProjectUptimeSubscriptio
         # Resolve the issue so that we don't see it in the ui anymore
         resolve_uptime_issue(uptime_monitor)
 
+    uptime_subscription = uptime_monitor.uptime_subscription
+
     uptime_monitor.update(
         status=ObjectStatus.DISABLED,
         # We set the status back to ok here so that if we re-enable we'll start from a good state
         uptime_status=UptimeStatus.OK,
     )
-    quotas.backend.disable_seat(DataCategory.UPTIME, uptime_monitor)
 
-    uptime_subscription = uptime_monitor.uptime_subscription
+    if detector := get_detector(uptime_monitor.uptime_subscription):
```

Comment:
Same thing for transaction here

Reply:
Same as my comment here https://github.com/getsentry/sentry/pull/89411/files?w=1#r2040301502